src/modules/img_thumb_gen/scripts/fix_missing_thumbs.py

```diff
@@ -23,7 +23,7 @@
 
 def generate_thumbnails(data, context):
   # Check if the file format is correct
-  if not data['name'].endswith(('jpg', 'jpeg', 'JPG', 'JPEG')):
+  if not data['name'].lower().endswith(('jpg', 'jpeg')):
```

Comment:
great. Sry, should have mentioned to keep the '.' as part of the file extension (".jpg", ".jpeg").